index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- navbar starts -->
+    
+    <header class="navbar">
+        <section class="logo">
+            <a href="#">My Logo</a>
+        </section>
+        <section class="union-bar">
```

Comment:
- [ ] please like the previous reviewer noted, kindly surround or wrap your hamburger icon with a nav tag

* The `nav` tag is for semantics reason
    
```
<nav>
       <img src="./Properties/Union.png" alt="union-bar">
 </nav>

```